jaeger-jaxrs2/src/test/java/com/uber/jaeger/filters/jaxrs2/ServerFilterTest.java

```diff
@@ -97,8 +101,8 @@ public void filter() throws Exception {
     Span span = spans.get(0);
     Map<String, Object> tags = span.getTags();
 
-    assertTrue(span.isRPC() && !span.isRPCClient());
-    assertEquals(source, span.getPeer().getService_name());
+    assertTrue(ThriftSpanConverter.isRPC(span) && !ThriftSpanConverter.isRPCClient(span));
```

Comment:
maybe it's worth leaving these helper methods on the Span, they are kind of used all over the place.

(sigh, I miss Scala)